infor-design/enterprise
# 5609 - Add mobile enhancements and style changes

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR adds mobile enhancements and style changes to the cards component.
- Card should have `16px` padding on both sides (basic card and card variations)
- swipe action right button should have `5px` border-radius (top right, bottom right) see the [Figma](https://www.figma.com/file/2pHthwwOhchvnpFXp69O7H/IDS-Card?node-id=272%3A11075) design
- swipe action left button should have `5px` border-radius (top left, bottom left) see the [Figma](https://www.figma.com/file/2pHthwwOhchvnpFXp69O7H/IDS-Card?node-id=272%3A11075) design
- Swipe action (left and right) should be wider than before. See the [Figma](https://www.figma.com/file/2pHthwwOhchvnpFXp69O7H/IDS-Card?node-id=272%3A11075) design
- Swipe action icon should be `16px x 16px`
- The contents of it should always be centered
- The second line should be truncated

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/5609

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.) Test the basic card
- Go to http://localhost:4000/components/cards/example-index
- Check the `.card-header`
- It should have `16px` spaces on both sides

2.) Test the card variations
- Go to http://localhost:4000/components/cards/example-variations-hitboxes.html
- Card variations should have `16px` space on both sides

3.) Test the swipe action
- Go to http://localhost:4000/components/swipe-action/example-index.html
- Toggle the device toolbar in the chrome dev tool
- swipe action right button should have `5px` border-radius (top right, bottom right) see the [Figma](https://www.figma.com/file/2pHthwwOhchvnpFXp69O7H/IDS-Card?node-id=272%3A11075) design
- swipe action left button should have `5px` border-radius (top left, bottom left) see the [Figma](https://www.figma.com/file/2pHthwwOhchvnpFXp69O7H/IDS-Card?node-id=272%3A11075) design
- Swipe action (left and right) should be wider than before. See the [Figma](https://www.figma.com/file/2pHthwwOhchvnpFXp69O7H/IDS-Card?node-id=272%3A11075) design
- Swipe action icon should be `16px x 16px`
- The contents of it should always be centered
- The second line should be truncated

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
